x-pack/legacy/plugins/ml/public/services/ml_api_service/jobs.js

```diff
@@ -134,4 +134,77 @@ export const jobs = {
       method: 'GET',
     });
   },
+
+  newJobLineChart(
```

Comment:
could we just pass a `data` object then we wouldn't have to list all the args/properties twice? (guess this might make sense once migrating to typescript because then the `data` object could be defined by an interface instead of having single functions args here)

Reply:
This can be addressed when we typescriptify the `ml_api_service` file.